src/adtl/autoparser/openai_calls.py

```diff
@@ -0,0 +1,120 @@
+"Contains all functions that call OpenAI's API."
+
+from __future__ import annotations
+
+from openai import OpenAI
+
+from .util import ColumnDescriptionRequest, MappingRequest, ValuesRequest
+
+
+def _get_definitions(
```

Comment:
Not a blocker - but perhaps worth rewriting to use baseclasses and inheritance since each LLM would have the same set of functions